### PR TITLE
[FrameworkBundle] Fix wrong merge

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | --
| License       | MIT

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php [hunk 1]
```diff
@@ -124,9 +124,6 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $io = new SymfonyStyle($input, $output);
-        $errorIo = $output instanceof ConsoleOutputInterface ? new SymfonyStyle($input, $output->getErrorOutput()) : $io;
```
Comment (OskarStark) [inline src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php:128]: Check some lines below, `$io` is initialized again

Comment (chalasr) [reply src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php:128]: Won't the command break at l128? I guess the second io initialization below is the one that should be removed instead

Comment (OskarStark) [reply src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php:128]: You could be right, not tested

Comment (OskarStark) [reply src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php:128]: I think we should move the lower one up

**Review by chalasr — APPROVED** [review]

**Comment by OskarStark** [comment]
@chalasr I also removed the caution about the deprecation, as we are on 6.4 branch

**Comment by chalasr** [comment]
Indeed 👍

**Review by xabbuh — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @OskarStark.

**Pull request merged**